api/types.proto

```diff
@@ -30,12 +30,27 @@ message Annotations {
 	repeated IndexEntry indices = 4 [(gogoproto.nullable) = false];
 }
 
+message Set {
+	message Void {}
+	map<string, Void> val = 1;
```

Comment:
In golang I think the normal practice is `struct{}{}`? Do we need to create `Void` here?